storage/storage.cpp

```diff
@@ -764,35 +769,6 @@ TStatus Storage::CountryStatusFull(TCountryId const & countryId, TStatus const s
   return TStatus::EOnDisk;
 }
 
-MapOptions Storage::NormalizeDownloadFileSet(TCountryId const & countryId, MapOptions options) const
```

Comment:
As far as I understand the method is needed only if we're downloading two components mwm. But we are not planning to download them with new downloader.